webportal/config/webpack.common.js

```diff
@@ -163,6 +163,10 @@ const config = {
       jQuery: 'jquery',
       'window.jQuery': 'jquery'
     }),
+    new webpack.ProvidePlugin({
```

Comment:
Haven’t we used this cookies plugin before?

Reply:
not global